index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <title>Calculator</title>
+  </head>
+  <body>
+    <div class="calc">
+      <div class="calc__display">
+        <input disabled type="text" id="display" value="0" name="display" class="calc-display-input"/>
+      </div>
+      <div class="calc__button">
+        <button class="clear" id="c">c</button>
+        <button class="clear" id="ce">ce</button>
+        <button class="operation-style operation">/</button>
+        <button class="operation-style operation">*</button>
+        <button class="number-style number">7</button>
+        <button class="number-style number">8</button>
+        <button class="number-style number">9</button>
+        <button class="operation-style operation">-</button>
+        <button class="number-style number">4</button>
+        <button class="number-style number">5</button>
+        <button class="number-style number">6</button>
+        <button class="operation-style operation">+</button>
+        <button class="number-style number">1</button>
+        <button class="number-style number">2</button>
+        <button class="number-style number">3</button>
```

Comment:
You don't have to separate classes `number-style` for css and `number` for js. You can use only one class for both cases. The same is for `operation-style` and `operation`.

Reply:
DONE!